authors/apps/tests/test_reporting_article.py

```diff
@@ -0,0 +1,63 @@
+from .base import BaseTestCase
+from rest_framework import status
+from django.urls import reverse
+
+from authors.apps.profiles.models import Notification
+from . import (user_login, post_article, report_data, invalid_report_data_int,
+               invalid_report_data_blank)
+from ..article.utils import invalid_string
+
+
+class NotificationTests(BaseTestCase):
```

Comment:
@arthurarty  Good work done.  You should also consider adding a docstring to this class.

Reply:
done. thanks for catching that.